src/custom/components/ArrowWrapperLoader/index.tsx

```diff
@@ -63,7 +65,7 @@ export const Wrapper = styled.div<ShowLoaderProp>`
   > div > img {
     height: 100%;
     width: 100%;
-    padding: 2px 2px 0;
+    padding: ${({ noPadding }) => (noPadding ? `0px` : `2px 2px 0`)};
```

Comment:
curious why we need this? u end up leaving the nose picking GIF right?

Reply:
Good catch, reverted this also